docs/csharp/language-reference/compiler-messages/cs8515.md

```diff
@@ -0,0 +1,43 @@
+---
+description: "Compiler Error CS8515"
+title: "Compiler Error CS8515"
+ms.date: 24/10/2022
+f1_keywords:
+  - "CS8515"
+helpviewer_keywords:
+  - "CS8515"
+---
+# Compiler Error CS8515
+
+Parentheses are required around the switch governing expression.
+
+
+## Example
+
+ The following sample generates CS8515:
+
+```csharp
+// CS8515.cs
+
+int x = 5;
+switch (x)
+    case 5:
+        break;
```

Comment:
The example you've used generates a number of errors and warnings. I suggest the following to isolate only CS8515:

```suggestion
int x = 5;
switch x
{
    case 5:
        break;
}
```